_ont/ont-zyxel-pmg3000-d20b.md

```diff
@@ -76,7 +76,7 @@ set sn ALCLf0f0f0f0
 {% include alert.html content="the PLOAM is stored in the ASCII format." alert="Note"  icon="svg-info" color="blue" %}
 This can be done easily via web ui. If you prefer to do it via the shell use:
 ```sh
-manufactory
+hal
```

Comment:
Just to have a history why this modification?

Reply:
'set password' is not valid for manufactory, exists only in hal

Reply:
perfect